### DISCUSSION TITLE
Postgres Connection Failure  pga_hba.co... [Jul 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2995] [Discord Threads]
> 🙋‍♂️ **jeanbonbeur**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@996487497439781014>    
#### If you are mac/linux- you can try host.docker.internal instead of localhost/127.0.0.1  
#  
> 👨‍💻 **pranavxc**  
#### Or alternately you can specify network as `host` in docker compose  --- https://docs.docker.com/compose/compose-file/compose-file-v3/#network_mode   
####    
#### this way all ports will be available within docker container  
#  
> 👨‍💻 **pranavxc**  
#### ```yml   
#### version: '2.1'   
####    
#### services:   
####   nocodb:   
####     image: nocodb/nocodb:latest   
####     network_mode: "host"   
####     restart: always   
####     volumes:   
####       - nc_data:/usr/app/data   
####     environment:   
####       NC_DB: "pg://localhost:5432?u=myuser&p=mypassword&db=mydb"   
#### ```  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### Perfect, thank you for your answers.   
####    
#### I fill in the file /nocodb/docker-compose/pg/docker-compose.yml the following values:   
####    nocodb:   
####      depends_on:   
####        root_db:   
####          condition: service_healthy   
####      image: nocodb/nocodb:latest   
####      network_mode: "host"   
####      ports:   
####        - "8080:8080"   
####      restart: always   
####      volumes:   
####        - nc_data:/usr/app/data   
####      environment:   
####        NC_DB: "pg://localhost:5432?u=myuser&p=mypassword&d=mydb"   
####    
####     
#### I run the following cmd: cd nocodb/docker-compose/pg/docker-compose.yml; docker-compose up -d --build   
####    
#### I reload my access page to the application. I connect to an existing database and I test the connection = success   
####    
#### Thanks a lot for your help !  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1000467955181690980)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
